### PR TITLE
Fix and update event related UG components

### PR DESCRIPTION
**Pull request by LinkedInk**

### CODE REVIEW

**Review by bangyiwu — DISMISSED** [review]
lgtm

File: docs/UserGuide.md [hunk 1]
```diff
@@ -254,7 +254,13 @@ Examples:
...
 Adds an event to the event list.
 
-Format: `add -e d/DESCRIPTION at/DATE_TIME`
+Format: `add -e [d/DESCRIPTION] [at/DATE_TIME] [ap/CONTACT_INDEX_LIST]`
+
+1) `DESCRIPTION` = What you want the description of the event to be.
+1) `DATE_TIME` = Date and time of the event.
+1) `CONTACT_INDEX_LIST` = Indexes of contacts in the middle panel you want to add to the event.
+`*` is accepted as a way to add all the contacts from the panel. (See examples below)
+1) `[ap/CONTACT_INDEX_LIST]` is optional and can be omitted.
```
Comment (fyshhh) [inline docs/UserGuide.md]: this documentation style seems inconsistent with the rest of the UG.

Comment (LinkedInk) [reply docs/UserGuide.md]: this has been fixed

File: docs/UserGuide.md [hunk 2]
```diff
@@ -254,7 +254,12 @@ Examples:
 
 Adds an event to the event list.
 
-Format: `add -e d/DESCRIPTION at/DATE_TIME`
+Format: `add -e d/DESCRIPTION at/DATE_TIME [ap/CONTACT_INDEX_LIST]`
+
+* `DESCRIPTION` = What you want the description of the event to be.
+* `DATE_TIME` = Date and time of the event.
+* `CONTACT_INDEX_LIST` = Indexes of contacts in the middle panel you want to add to the event.
```
Comment (fyshhh) [inline docs/UserGuide.md]: i meant the use of "=" is inconsistent.

Comment (LinkedInk) [reply docs/UserGuide.md]: oh.... ok

Comment (LinkedInk) [reply docs/UserGuide.md]: fixed

**Review by bangyiwu — APPROVED** [review]
lgtm

**Comment by LinkedInk** [comment]
Previous update was minor, "the event" was changed to "event 1" under edit event examples.

**Pull request merged**